.examples/docker-compose/opensocial.at/README.md

```diff
@@ -0,0 +1,50 @@
+# Opensocial.at setup
+
+This example of the current opensocial.at configuration has to be seen as a possible "production-ready" environment.
+The focus of this configuration is on performance and scalability.
+
+## Prerequisites
+
+This setup needs some configuration first to be fully usable.
+
+1. It uses an external, dedicated database, which is not included here (you can just add a `mariadb` service directly)
+2. avatar caching needs to be enabled
+   1. Enable the system-config `system.avatar_cache`
+   2. Set `avatar_cache_path` to `/var/www/avatar`
+3. It uses a traefik docker service as overall reverse proxy for the whole docker environment
+   1. Otherwise, adaptations of the two services `web` and `avatar` are necessary 
+
+## The setup
+
+The setup splits Friendica in as much (micro)services as possible.
+
+### Split Frontend & Daemon
+
+This setup splits the frontend services from the background Daemon.
+So it's possible to scale different aspect from the frontend without harming states of the cronjob forks of the Daemon.  
```

Comment:
```suggestion
This setup splits the frontend services from the background daemon so that it's possible to scale the different aspects of the frontend without harming the state of the cronjob forks of the daemon.  
```